Makefile

```diff
@@ -9,6 +9,9 @@ SRC=$(shell find . -name '*.go')
 all: test
 
 deps:
+	go get github.com/jteeuwen/go-bindata/...
+	go get github.com/elazarl/go-bindata-assetfs/...
+	go generate
```

Comment:
Must be before `go get -t ./...` or the go get fails because there's no .go file in the assets dir